backend/src/main/java/com/woowacourse/gongseek/comment/application/CommentService.java

```diff
@@ -0,0 +1,76 @@
+package com.woowacourse.gongseek.comment.application;
+
+import com.woowacourse.gongseek.article.domain.Article;
+import com.woowacourse.gongseek.article.domain.repository.ArticleRepository;
+import com.woowacourse.gongseek.auth.presentation.dto.LoginMember;
+import com.woowacourse.gongseek.comment.domain.Comment;
+import com.woowacourse.gongseek.comment.domain.repository.CommentRepository;
+import com.woowacourse.gongseek.comment.presentation.dto.CommentRequest;
+import com.woowacourse.gongseek.comment.presentation.dto.CommentResponse;
+import com.woowacourse.gongseek.member.domain.Member;
+import com.woowacourse.gongseek.member.domain.repository.MemberRepository;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+@RequiredArgsConstructor
+@Service
```

Comment:
```suggestion
@RequiredArgsConstructor
@Transactional
@Service
```
어노테이션 순서를 중요도가 낮을수록 클래스랑 멀어지게 두는 컨벤션이 좋을것 같아요!

Reply:
넵 생성자 어노테이션은 가장 위로 올릴게용